index.js

```diff
@@ -480,16 +480,31 @@ function multilineStyle(textarea: HTMLTextAreaElement, arg: StyleArgs) {
 
 function orderedList(textarea: HTMLTextAreaElement): SelectionRange {
   const orderedListRegex = /^\d+\.\s+/
+  const noInitialSelection = textarea.selectionStart === textarea.selectionEnd
   let selectionEnd
   let selectionStart
   let text = textarea.value.slice(textarea.selectionStart, textarea.selectionEnd)
+  let textToUnstyle = text
   let lines = text.split('\n')
-
-  const undoStyling = lines.every(line => orderedListRegex.test(line))
+  let startOfLine, endOfLine
+  if (noInitialSelection) {
+    const linesBefore = textarea.value.slice(0, textarea.selectionStart).split(/\n/)
+    startOfLine = textarea.selectionStart - linesBefore[linesBefore.length - 1].length
+    endOfLine = wordSelectionEnd(textarea.value, textarea.selectionStart, true)
+    textToUnstyle = textarea.value.slice(startOfLine, endOfLine)
```

Comment:
Fix for 2.

When there is no selection range, get the range for the **current line** and check if it can be unstyled. This is a separate variable because we want to keep `text` as it was if `!undoStyling`.